api/pkg/cli/cmd/install/install_test.go

```diff
@@ -543,3 +584,21 @@ func TestInstall_DeprecatedVersion(t *testing.T) {
 	assert.Equal(t, "WARN: tekton pipelines version unknown, this resource is compatible with pipelines min version v0.12\nWARN: This version has been deprecated\nTask foo(0.2) installed in hub namespace\n", buf.String())
 	assert.Equal(t, gock.IsDone(), true)
 }
+
+func prepareOpts(t *testing.T, buf *bytes.Buffer, hubType, from, version string) *options {
```

Comment:
Should we rename this function to something like `createOpts` or `constructOpts`

Reply:
SGTM. Updated to `createOpts `!